src/log-error.js

```diff
@@ -21,7 +21,8 @@ export function logError(error) {
     if (window.onerror) {
         // set timeout since jasmine doesn't expect window.onerror to be called from its own context
         setTimeout(() => {
-            window.onerror(getErrorObject(error));
+            const errObj = getErrorObject(error);
+            window.onerror(errObj.message, errObj.stack, 0, 0, errObj);

```

Comment:
Docs for onerror mention that second argument should be string url to a file. Is it what's inside errObj.stack?

Reply:
I will fix this